components/mpas-seaice/src/shared/mpas_seaice_icepack.F

```diff
@@ -6548,6 +7007,13 @@ subroutine init_column_tracer_object_parent_indices(domain, tracerObject)
        enddo ! iSnowLayer
     endif
 
+    ! Floe size distribution
+    if (config_use_floe_size_distribution) then
+       do iFloeSize = 1, nFloeCategories
+          tracerObject % parentIndex(tracerObject % index_floeSizeDist + iFloeSize - 1) = 1 ! WHAT VALUE SHOULD THIS BE SET TO ????
```

Comment:
I think this should be 0.  This designation indicates which quantity the tracer is being carried on or in.  In this case, the FSD is a tracer on ice area (so 0).  Snow characteristics in the code just above this are carried around in the snow volume (so 2).  The tracer design is pretty slick (makes advection easy) but it's a lot of bookkeeping.  How it works is described here, using melt ponds as an example: https://cice-consortium-icepack.readthedocs.io/en/main/science_guide/sg_tracers.html

Reply:
this is very helpful to understand the tracer stuff - thanks!